src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -538,6 +548,49 @@ public void shootAtPlayer(int across, int down) {
 
     }
 
+    public void easyComputerFire() {
```

Comment:
The code in this function is very clean. I would say that you may want to be more consistent on your spacing, and reduce the amount a little. The code is a little too spread out.